### ISSUE TITLE
Unit Frames are off-center

### ISSUE THREAD

**Issue by anthonypeyton** [DakJaniels/LuiExtended#88]
I've reinstalled the addon  (ensured folder was deleted) three times trying to resolve this. On the last try I even disabled all other addons in an attempt to troubleshoot. The odd part is this only affect the unit frames, not any of the others. 

![screenshot_20180710_185536](https://user-images.githubusercontent.com/30835325/42542121-5ac58f5a-8473-11e8-8a1b-c18f57851e07.png)

![screenshot_20180710_190022](https://user-images.githubusercontent.com/30835325/42542154-8420a0ba-8473-11e8-9d9c-f746523210ad.png)

**Comment by ArtOfShred** [comment]
Hmm, I'll do a comparison for what I'm anchoring unit frames to vs the other components and see if I can fix this to work better on larger displays.

**Comment by anthonypeyton** [comment]
Let me know how I can help. I'm pretty comfortable editing XML and JSON but don't know what I'm looking for here. I have two ultrawides which I can use for testing, 3440x1440 and 2560x1080.

**Comment by ArtOfShred** [comment]
Ah I see, Buff & Debuff containers are anchored to the default player HP bar, whereas the unitFrames just have coordinate positions.

I just updated the UnitFrame positions to anchor off the center of the screen instead of the Left.

Additionally I added an option to the UnitFrames menu that allows you to select your Resolution in order to change the default positions for Custom Unit Frames: https://puu.sh/BYAEs.png

If you get a chance, I updated the GitHub build with these changes - see if the 4K one is decently positioned for you, I tried to do the math reasonably well.

I'll mark this as resolved later if everything checks out.
